picasso-sample/src/main/java/com/example/picasso/SampleContactsActivity.java

```diff
@@ -30,6 +34,9 @@
     implements LoaderManager.LoaderCallbacks<Cursor> {
   private SampleContactsAdapter adapter;
 
+  private static final String sReadContacts = Manifest.permission.READ_CONTACTS;
+  private static final int sReadContactsCode = 100;
```

Comment:
Is it picasso's style ? It's different from [AOSP](https://source.android.com/source/code-style).

Ok I will change it.